charts/pravega/README.md

```diff
@@ -38,15 +38,42 @@ The following table lists the configurable parameters of the Pravega chart and t
 
 | Parameter | Description | Default |
 | ----- | ----------- | ------ |
-| `version` | Version for Pravega cluster | `0.5.0` |
-| `zookeeperUri` | Zookeeper service address | `zk-client:2181` |
-| `externalAccess.enabled` | Enable Pravega external access | `false` |
-| `externalAccess.type` | Pravega external access type | `LoadBalancer` |
-| `bookkeeperUri` | URL for Bookkeeper Servers | `bookkeeper-client:3181` |
-| `pravega.image.repository` | Image repo for Pravega image | `pravega/pravega` |
-| `pravega.controllerReplicas` | Replicas for controller | `1` |
-| `pravega.segmentStoreReplicas` | Replicas for segmentStore | `1` |
-| `pravega.debugLogging` | Enable debug logging | `false` |
-| `pravega.cacheVolumeRequest` | Request storage for cacheVolume | `20Gi` |
-| `pravega.storage.className` | Storage class for Pravega volume | `standard` |
-| `pravega.tier2` | Name of the PVC used for Tier 2 storage | `pravega-tier2` |
+| `version` | Pravega version | `0.5.0` |
```

Comment:
can we change the pravega version to `0.7.0`

Reply:
done